### ISSUE TITLE
The new rh6 and rh7 egg data folders need to be added to setup.py

### ISSUE THREAD

**Issue by kihneman** [enthought/okonomiyaki#345]
The tests in the buildsystem don't pass without the egg data included in the setup.py

**Comment by kihneman** [comment]
Here is a traceback for failing tests in the buildsystem (https://github.com/enthought/buildsystem/pull/2591):
```
======================================================================
ERROR: test_support_lower_compatible_version (okonomiyaki.file_formats.tests.test__egg_info.TestEggMetadata)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/workspace/Buildsystem/test/.r/envs/build-env/lib/python2.7/site-packages/okonomiyaki/file_formats/tests/test__egg_info.py", line 1237, in test_support_lower_compatible_version
    metadata = EggMetadata.from_egg(egg)
  File "/home/vagrant/workspace/Buildsystem/test/.r/envs/build-env/lib/python2.7/site-packages/okonomiyaki/file_formats/_egg_info.py", line 722, in from_egg
    return cls._from_egg(path_or_file, sha256, strict)
  File "/home/vagrant/workspace/Buildsystem/test/.r/envs/build-env/lib/python2.7/site-packages/okonomiyaki/file_formats/_egg_info.py", line 777, in _from_egg
    with zipfile2.ZipFile(path_or_file) as zp:
  File "/home/vagrant/workspace/Buildsystem/test/.r/envs/build-env/lib/python2.7/site-packages/zipfile2/_zipfile.py", line 59, in __init__
    super(ZipFile, self).__init__(file, mode, compression, allowZip64)
  File "/home/vagrant/workspace/Buildsystem/test/.r/envs/build-env/lib/python2.7/zipfile.py", line 756, in __init__
    self.fp = open(file, modeDict[mode])
IOError: [Errno 2] No such file or directory: '/home/vagrant/workspace/Buildsystem/test/.r/envs/build-env/lib/python2.7/site-packages/okonomiyaki/utils/test_data/eggs/rh6_x86_64/nose-1.3.4-1.egg'
```

**Comment by itziakos** [comment]
fixed by #346
